crates/hir/src/lib.rs

```diff
@@ -266,6 +266,10 @@ impl Crate {
         let data = &db.crate_graph()[self.id];
         data.potential_cfg_options.clone().unwrap_or_else(|| data.cfg_options.clone())
     }
+
+    pub fn id(&self) -> CrateId {
+        self.id
+    }
```

Comment:
There are `From` impls to go from hir type to id, no need for this

Reply:
removed!